docs/dev/analysis/features/cht-axis-title.rst

```diff
@@ -0,0 +1,231 @@
+    ...
+    AttributeError: 'NoneType' object has no attribute 'text_frame'
+    >>> chart.category_axis.has_title = True
+    >>> chart.category_axis.title.text_frame.text = 'Category Axis Title'
+    >>> chart.category_axis.title.text_frame.text
+    u'Category Axis Title'
+
+
+    >>> chart.value_axis.has_title
+    False
+    >>> chart.value_axis.title.text_frame.text = 'Value Axis Title'
+    ...
+    AttributeError: 'NoneType' object has no attribute 'text_frame'
+    >>> chart.value_axis.has_title = True
+    >>> chart.value_axis.title.text_frame.text = 'Value Axis Title'
+    >>> chart.value_axis.title.text_frame.text
+    u'Value Axis Title'
+
+
+The axis titles use a TextFrame. Only the "text" property has been
+reviewed and known to work for the axis titles. Some of the other
+properties may function as well but only the text property has been
+confirmed to work.
+
+
+Related Schema Definitions
+--------------------------
+
+.. highlight:: xml
+
+::
+
+  <xsd:group name="EG_AxShared">
+    <xsd:sequence>
+      <xsd:element name="axId"           type="CT_UnsignedInt"/>
+      <xsd:element name="scaling"        type="CT_Scaling"/>
+      <xsd:element name="delete"         type="CT_Boolean"           minOccurs="0"/>
+      <xsd:element name="axPos"          type="CT_AxPos"/>
+      <xsd:element name="majorGridlines" type="CT_ChartLines"        minOccurs="0"/>
+      <xsd:element name="minorGridlines" type="CT_ChartLines"        minOccurs="0"/>
+      <xsd:element name="title"          type="CT_Title"             minOccurs="0"/>
+      <xsd:element name="numFmt"         type="CT_NumFmt"            minOccurs="0"/>
+      <xsd:element name="majorTickMark"  type="CT_TickMark"          minOccurs="0"/>
+      <xsd:element name="minorTickMark"  type="CT_TickMark"          minOccurs="0"/>
+      <xsd:element name="tickLblPos"     type="CT_TickLblPos"        minOccurs="0"/>
+      <xsd:element name="spPr"           type="a:CT_ShapeProperties" minOccurs="0"/>
+      <xsd:element name="txPr"           type="a:CT_TextBody"        minOccurs="0"/>
+      <xsd:element name="crossAx"        type="CT_UnsignedInt"/>
+      <xsd:choice minOccurs="0" maxOccurs="1">
+        <xsd:element name="crosses"   type="CT_Crosses"/>
+        <xsd:element name="crossesAt" type="CT_Double"/>
+      </xsd:choice>
+    </xsd:sequence>
+  </xsd:group>
+
+  <xsd:complexType name="CT_Title">
+    <xsd:sequence>
+      <xsd:element name="tx" type="CT_Tx" minOccurs="0" maxOccurs="1"/>
+      <xsd:element name="layout" type="CT_Layout" minOccurs="0" maxOccurs="1"/>
+      <xsd:element name="overlay" type="CT_Boolean" minOccurs="0" maxOccurs="1"/>
+      <xsd:element name="spPr" type="a:CT_ShapeProperties" minOccurs="0" maxOccurs="1"/>
+      <xsd:element name="txPr" type="a:CT_TextBody" minOccurs="0" maxOccurs="1"/>
+      <xsd:element name="extLst" type="CT_ExtensionList" minOccurs="0" maxOccurs="1"/>
+    </xsd:sequence>
+  </xsd:complexType>
```

Comment:
I usually line these up into columns for quick scanning. Also, the maxOccurs="1" attributes can come out as that's the default for xsd:element; cuts down the noise a bit.

Also, schema excerpts go at the very end. XML specimens come prior. Can't remember why exactly, but it's handy to whip down to the bottom with the mouse when you need the schema for reference during implementation :)